src/models/neural_networks/ealstm.py

```diff
@@ -103,32 +115,38 @@ def _initialize_model(self, x_ref: Optional[Tuple[torch.Tensor, ...]]) -> nn.Mod
                       experiment=self.experiment,
                       yearly_agg_size=self.yearly_agg_size,
                       current_size=self.current_size,
-                      include_latlons=self.include_latlons)
+                      include_latlons=self.include_latlons,
+                      static_size=self.static_size)
 
 
 class EALSTM(nn.Module):
     def __init__(self, features_per_month, dense_features, hidden_size,
                  rnn_dropout, include_latlons, include_pred_month,
-                 experiment, yearly_agg_size=None, current_size=None):
+                 experiment, yearly_agg_size=None, current_size=None,
+                 static_size=None):
         super().__init__()
 
         self.experiment = experiment
         self.include_pred_month = include_pred_month
         self.include_latlons = include_latlons
         self.include_yearly_agg = False
+        self.include_static = False
 
-        assert include_latlons or (yearly_agg_size is not None), \
-            "Need one of latlons or yearly mean for the static input"
-        static_size = 0
+        assert include_latlons or (yearly_agg_size is not None) or (static_size is not None), \
+            "Need at least one of {latlons, yearly mean, static} for the static input"
```

Comment:
so are `latlons // yearly_mean` not percieved as `static` yet because they are derivatives of the dynamic variables? Would it make sense to save a `static` file of these into the `interim/static` folder so its absolutely clear the data being used? Or is that overkill and wasting storage?

Reply:
Yea - because they are calculated from the dynamic variables (just before begin used), it would be hard to save them in the static folder

I think if we document the dataloader well it should be sufficient?